classes/app/helper/times/factory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace assignsubmission_pxaiwriter\app\helper\times;
+
+
+use assignsubmission_pxaiwriter\app\interfaces\factory as base_factory;
+use DateTime;
+use DateTimeZone;
+
+/* @codeCoverageIgnoreStart */
+defined('MOODLE_INTERNAL') || die();
+/* @codeCoverageIgnoreEnd */
+
+class factory implements interfaces\factory
+{
+    private base_factory $factory;
+
+    public function __construct(base_factory $factory)
+    {
+        $this->factory = $factory;
+    }
+
+    public function current_timestamp(): int
+    {
+        return time();
```

Comment:
"timestamp" is a fairly broad term. I think it would be better to rename this function to something like `unix_timestamp()`, to better indicate which output is expected

Reply:
Call it ``unix_timestamp()`` doesn't not help to solve the naming convention. Because it return the current time in the unix timestamp.
Maybe ``current_time()`` will be enough in this context.